package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "website-frontend",
+  "version": "1.0.0",
+  "description": "UI frontend repository for the Providence Geeks website. Built with React and webpack.",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "webpack -p",
```

Comment:
This looks good, I would recommend removing `src/**/*/*.js` from the pattern, since those files should be covered by **webpack** when it builds